fuzzing/nxt_basic_fuzz.c

```diff
@@ -81,11 +159,78 @@ nxt_time_fuzz(const u_char *data, size_t size)
 }
 
 
+void
+nxt_uri_fuzz(const u_char *data, size_t size)
+{
+    u_char  *dst;
+
+    dst = nxt_zalloc(size * 3);
+    if (dst == NULL) {
+        return;
+    }
+
+    nxt_decode_uri(dst, (u_char *) data, size);
+    nxt_decode_uri_plus(dst, (u_char *) data, size);
+
+    nxt_encode_uri(NULL, (u_char *) data, size);
+    nxt_encode_uri(dst, (u_char *) data, size);
```

Comment:
Again I would prefer no space after the casts...

Reply:
fair enough.